### DISCUSSION TITLE
Obsidian Publish Alternative, How to One-click Upload Your Notes on Your Own Website - Mathis Gauthey's Website

### DISCUSSION THREAD

**Discussion by giscus** [mathisgauthey/mathisgauthey.github.io#1] [_]
# Obsidian Publish Alternative, How to One-click Upload Your Notes on Your Own Website - Mathis Gauthey's Website

Obsidian Publish Alternative, How to One-click Upload Your Notes on Your Own Website This post is an extension of my main post showcase of my Obsidian workflow. You might want to check it out as well.
Create You Own Hugo Website Prerequisites I&rsquo;ll assume that you installed Hugo following their quick-start documentation and Git on your computer, and that you know how to clone a repository from Github. If not, Google and Chat GPT are your friends here.

https://mathisgauthey.github.io/obsidian-publish-alternative-how-to-one-click-upload-your-notes-on-your-own-website/

**Comment by 12risobo** [comment]
Hi Mathis,

Trying to work through your guide but stuck at 1.3 at 
*You will now come back to your Documents folder and git clone your previously created repository. You now have a website_name folder containing your Hugo site folder with the theme as a git submodule. And a website_name.github.io folder containing a README.md file.*

This is not clear to me. With clone your previously created repository, what do you mean by that? 

I did try to clone the subfolder / repository names pekoma and I got the following error:
$ git clone /pekoma
fatal: destination path 'pekoma' already exists and is not an empty directory.

So it seems I try to clone it but I already have the folder. 

What am I doing wrong? It would be great if you can explain this to me.

**Reply by mathisgauthey** [reply]
Hey there ! I think I could do better with steps like that :
1. Create your repository **online** on GitHub, naming it `website_name.github.io`
2. Follow the step to create a local Hugo website folder named `website_name` with a theme as a git submodule. Don't forget to copy the content of the `exempleSite` folder to the root of your local website folder.
3. Clone your online repository. You now have your online repository `website_name.github.io` on local and your local website `website_name`.
4. Copy the content of your local website `website_name` to your repository, commit, and push !

Should be alright then, I merely rephrased but it could help ✌️ Tell me if it went right !
